MANIFEST.in

```diff
@@ -1,2 +1,8 @@
-include versioneer.py
-include aehmc/_version.py
+prune .github
+prune tests
```

Comment:
You may wish to consider leaving in `tests`. Although they aren't required for runtime execution, they don't take up much space and it is useful for package maintainers to have them. For example, see https://github.com/pypa/setuptools_scm/issues/806#issuecomment-1442254345